cmd/open.go

```diff
@@ -16,5 +16,27 @@ func (h *Handler) Open(ctx context.Context, req *entity.CommandRequest) error {
 		return err
 	}
 
+	if (req.Cmd.Use != "open") {
+		return h.ctrl.OpenProjectPathInBrowser(ctx, projectId, environmentId, req.Cmd.Use)
```

Comment:
If someone does `railway open potato`, it will simply open their dashboard. Is that okay or do we want to block unknown subcommand?

Reply:
I think it would make sense to show help for open command in this case